circle.yml

```diff
@@ -8,7 +8,14 @@ machine:
 
 ## Customize deployment commands
 deployment:
-  staging:
+  production:
+    branch: master
+    commands:
+      - heroku pg:backups  --app vote-kompot
+      - git push git@heroku.com:vote-kompot.git $CIRCLE_SHA1:refs/heads/master
+      - heroku run rake db:migrate --app vote-kompot
+deployment:
```

Comment:
I'm pretty sure that duplicated section `deployment → production` shouldn't be there (and most probably a mistake).